src/handlers/signup.handler.ts

```diff
@@ -33,6 +33,7 @@ export class SignUpHandler extends MessageHandler {
       id: uuidv4(),
       name,
       type,
+      spells: [],
```

Comment:
If the player is an `UNITY` player we should set his speed to `DEFAULT_PLAYER_SPEED`